README.md

```diff
@@ -73,12 +73,15 @@ Please note that the full path needs to be configured and env variables like $US
 
 xhyve:
 ```
+export CB_LOCAL_DEV_BIND_ADDR=192.168.64.1
+export PUBLIC_IP=192.168.64.2
```

Comment:
Please remove PUBLIC_IP and CB_LOCAL_DEV_BIND_ADDR exports. cloudbreak-depolyer setup these vars if you use "cbd" docker-machine name